README.md

```diff
@@ -588,6 +588,56 @@ crud.len('customers')
 ...
 ```
 
+### Count
```

Comment:
We need to add count() to a table above ("Table below describe what operations supports custom sharding key").

Reply:
Done.